### PR TITLE
[New] Add magnify@2.1.0 w/ git auto-update

### LINKED ISSUE #6604: [Request] Add Magnify
**Library name:** Magnify
**Git repository url:** https://github.com/thdoan/magnify
**npm package url(optional):** https://www.npmjs.com/package/magnify
**License(s):** https://github.com/thdoan/magnify/blob/master/LICENSE
**Official homepage:** http://thdoan.github.io/magnify/
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sashberd**
close #6604, cc @thdoan, using web-minify-helper for minifing files

Pull request for issue: #6604 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/thdoan/magnify/tree/master
 * Official website (optional, not the repository): https://thdoan.github.io/magnify/demo.html
 * NPM package url (optional): https://www.npmjs.com/package/magnify
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 8
   - Count of stars: 114
   - Count of forks: 30
 * NPM download stats (optional):
   - Downloads in the last day: 26
   - Downloads in the last week: 222
   - Downloads in the last month: 1038

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [X] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X] The parent of the commit(s) in the PR is not old than 3 days.
 * [X] Pull request is sending from a non-master branch with meaningful name.
 * [X] Separate unrelated changes into different commits.
 * [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [X] Close corresponding issue in commit message
 * [X] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/magnify/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  "version": "2.1.0",
+  "description": "A lightweight jQuery magnifying glass zoom plugin.",
+  "keywords": [
+    "jquery-plugin",
+    "jquery magnify",
+    "magnify",
+    "magnifying glass",
+    "zoom"
+  ],
+  "homepage": "http://thdoan.github.io/magnify/",
```
Comment (maruilian11) [inline ajax/libs/magnify/package.json]: Please use `https` instead.

**Comment by sashberd** [comment]
@maruilian11 The PR was updated

File: ajax/libs/magnify/package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        "files": [
+          "*.css"
+        ]
+      },
+      {
+        "basePath": "js",
+        "files": [
+          "*.js"
+        ]
+      }
```
Comment (maruilian11) [inline ajax/libs/magnify/package.json]: We prefer not to set `css` and `js` as basepath, could you turn to use:
```
{
  "basePath": "",
  "files": [
    "css/*.css",
    "js/*.js"
  ]
}
```

**Comment by sashberd** [comment]
@maruilian11 The PR was updated

**Comment by maruilian11** [comment]
v2.1.0 assets should also follow the auto-update config. Put files under `css` and `js`. Thanks.

**Comment by sufuf3** [comment]
ping @maruilian11 @extend1994 
Please help review this PR, thanks.

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by sashberd** [comment]
@PeterDaveHello I do not understand your the question. Could you be more clear?

**Comment by extend1994** [comment]
@sashberd `css/magnify.css` in this PR is different from the [upstream](https://github.com/thdoan/magnify/blob/v2.1.0/dist/css/magnify.css). Should find that previously, sorry!

**Comment by sashberd** [comment]
@extend1994 @PeterDaveHello The PR was updated. FYR

**Comment by sashberd** [comment]
@PeterDaveHello Hi man
I would like to talk with you inside gitter for a few minutes. Waiting for you there
Thanks

**Pull request merged**